posthog/warehouse/models/table.py

```diff
@@ -98,6 +98,10 @@ class TableFormat(models.TextChoices):
         help_text="Dict of all columns with Clickhouse type (including Nullable())",
     )
 
+    row_count: models.IntegerField = models.IntegerField(
```

Comment:
Is there a blocker in putting this count on the "external_data_source" that certain tables have connected to? row_count would be another field on the table model that is used for some (linked sources) and not for others (manual source)

Reply:
If we don't have a table for manual sources, how do we query them? As far as I'm aware, a table is required to store the S3 location of the parquet file, no?